integration-tests/gatsby-source-wordpress/test-fns/test-utils/queries.js

```diff
@@ -187,31 +187,35 @@ exports.queries = {
       isDynamic
       order
       originalContent
-      parentNode {
-        id
-        ... on WpPost {
-          title
-        }
-      }
+      # @todo this connection isn't working
```

Comment:
I'm guessing these todos are non-blocking?

Reply:
Indeed 👍 we need to fix them but not as part of this PR